extensions/elasticsearch/6.x/wazuh-template.json

```diff
@@ -415,6 +415,9 @@
                 "check.rationale": {
                   "type": "text"
                 },
+                "check.condition": {

```

Comment:
This field is inside the `oscap` block, it has no relation to SCA but the integration with OpenSCAP.